data/json/monstergroups.json

```diff
@@ -552,30 +553,24 @@
       { "monster" : "mon_zombie_grabber", "freq" : 35, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_grappler", "freq" : 5, "cost_multiplier" : 10 },
       { "monster" : "mon_zombie_hunter", "freq" : 10, "cost_multiplier" : 5 },
-      { "monster" : "mon_skeleton", "freq" : 30, "cost_multiplier" : 5 },
-      { "monster" : "mon_zombie_smoker", "freq" : 10, "cost_multiplier" : 5 },
+      { "monster" : "mon_zombie_smoker", "freq" : 10, "cost_multiplier" : 5, "starts" :   120 },
       { "monster" : "mon_zombie_shady", "freq" : 10, "cost_multiplier" : 4},
       { "monster" : "mon_zombie_shady", "freq" : 30, "cost_multiplier" : 15, "pack_size": [4,6]},
       { "monster" : "mon_zombie_gasbag", "freq" : 10, "cost_multiplier": 5},
       { "monster" : "mon_zombie_gasbag", "freq" : 3, "cost_multiplier": 10, "pack_size": [3,5]},
       { "monster" : "mon_zombie_swimmer", "freq" : 10, "cost_multiplier" : 5 },
       { "monster" : "mon_zombie_shrieker", "freq" : 30, "cost_multiplier" : 5 },
-      { "monster" : "mon_zombie_spitter", "freq" : 20, "cost_multiplier" : 5 },
+      { "monster" : "mon_zombie_spitter", "freq" : 20, "cost_multiplier" : 5, "starts" :   120 },
       { "monster" : "mon_zombie_acidic", "freq" : 30, "cost_multiplier" : 2 },
       { "monster" : "mon_zombie_electric", "freq" : 10, "cost_multiplier" : 5 },
-      { "monster" : "mon_zombie_necro", "freq" : 5, "cost_multiplier" : 25 },
       { "monster" : "mon_zombie_survivor", "freq" : 1, "cost_multiplier" : 25 },
```

Comment:
I'd say survivor zombies also shouldn't show up too early, since there hasn't been time for anyone to be a survivor

Reply:
Agree, will put survivor zombies on the 10 day timer (since they aren't all too dangerous and not too much stuff is being added yet.